src/app/components/compact-list/directives/contenteditable.directive.ts

```diff
@@ -0,0 +1,58 @@
+import {Directive, ElementRef, forwardRef} from "@angular/core";
+import {ControlValueAccessor, NG_VALUE_ACCESSOR, NG_VALIDATORS, FormControl} from "@angular/forms";
+
+@Directive({
+    selector: '[contenteditable]',
```

Comment:
Add another attribute selector here so that `contenteditable` is not overridden wherever it is used. Though I can't think off the top of my head where we'll use it again, it's still best practice. Overriding standard HTML attributes leads to confusion.